### PR TITLE
Add Korean texts to (semi-)auto edit list

### PR DESCRIPTION
**Pull request by revi**
I am regex newbie (despite editing regex for few years) so review needed for there, despite I think you aren't necessarily good at Korean. Or I can find someone who does Korean and regex.

### CODE REVIEW

File: app/config/semi_automated.yml [hunk 1]
```diff
@@ -91,6 +91,18 @@ parameters:
         Redirect:
             regex: 'Weiterleitung nach \[\[.*?erstellt'
             label: Weiterleitung
+    ko:
+        Undo:
+            regex: '판 편집을 되돌림'
+            tag: mw-undo
```
Comment (MusikAnimal) [inline app/config/semi_automated.yml]: This gets merged into the global rule, so the tag specification isn't needed here, same with rollback below

File: app/config/semi_automated.yml [hunk 2]
```diff
@@ -343,6 +355,23 @@ parameters:
         MoveToDraft:
             regex: '\[\[User:Evad37\/MoveToDraft\.js\|via script'
             link: User:Evad37/MoveToDraft
+    ko.wikipedia.org:
+        AutoWikiBrowser:
+            regex: 'using [[위키백과:AWB|AWB]]'
```
Comment (MusikAnimal) [inline app/config/semi_automated.yml]: Need to escape the brackets and pipes, so `'using \[\[위키백과:AWB\|AWB\]\]'`

File: app/config/semi_automated.yml [hunk 3]
```diff
@@ -343,6 +355,23 @@ parameters:
...
+            link: 위키백과:AWB
+            label: 오토위키브라우저
+        Popups:
+            regex: '[[백:팝업|NavPop]]'
```
Comment (MusikAnimal) [inline app/config/semi_automated.yml]: `'\[\[백:팝업\|NavPop\]\]'`

File: app/config/semi_automated.yml [hunk 4]
```diff
@@ -343,6 +355,23 @@ parameters:
...
+            link: 백:팝업
+            label: 내비게이션 팝업
+        Twinkle:
+            regex: '[[위키백과:도구/트윙클|TW]]'
```
Comment (MusikAnimal) [inline app/config/semi_automated.yml]: `'\[\[위키백과:도구\/트윙클\|TW\]\]'`

File: app/config/semi_automated.yml [hunk 5]
```diff
@@ -343,6 +355,23 @@ parameters:
...
+            link: 위키백과:도구/트윙클
+            label: 트윙클
+        WikiLove:
+            regex: '새 위키사랑 글'
+            tag: 'wikilove'
```
Comment (MusikAnimal) [inline app/config/semi_automated.yml:371]: I guess this tag should be moved to the global ruleset, but I'll take care of that.

We also need a page to link to. If there is no page, I guess let's link to `mw:Extension:WikiLove/ko`

**Review by MusikAnimal — CHANGES_REQUESTED** [review]

File: app/config/semi_automated.yml (hunk 1 above)
Comment (revi) [reply app/config/semi_automated.yml]: tag: isn’t correctly catching old (pre-tag era) edit summaries so I thought I would put that there.

Comment (MusikAnimal) [reply app/config/semi_automated.yml]: Correct, but any per-wiki or per-language rules get merged into global ones, e.g. https://github.com/x-tools/xtools/pull/181/files#diff-03baa01bde8a3fd8340f8f6d32967575R34

Adding it here won't hurt anything, though :) The merging thing is just a means to keep the code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Comment (revi) [reply app/config/semi_automated.yml]: OK, so I think `tag:` is the only thing that needs to be removed?

File: app/config/semi_automated.yml (hunk 5 above)
Comment (revi) [reply app/config/semi_automated.yml:371]: For Korean Wikipedia, I think `ko:위키백과:위키사랑` is short but better than almost-not-translated `mw:Extension:WikiLove/ko`.

**Comment by MusikAnimal** [comment]
There are a few other tiny issues, but I can take it from here. Many thanks!

**Pull request merged**